README.md

```diff
@@ -1 +1,76 @@
 # Multiplane optical physiology processing pipeline
+
+
+# aind-multiplane-ophys-pipeline
+
+The multiplane-ophys pipeline is a processing pipeline which uses [Suite2p](https://github.com/MouseLand/suite2p) for both motion correction and segmentation. The final outputs of the pipeline are ROI events detected by [OASIS](https://github.com/j-friedrich/OASIS).
```

Comment:
I would emphasize that this is for data in which multiple planar fields of view are collected simultaneously and maybe link to the mesoscope docs.

Reply:
You could also link to the single-plane pipeline here.